extensions/content-manager/requirements.txt

```diff
@@ -0,0 +1,4 @@
+# requirements.txt auto-generated by Posit Publisher
+# using /Users/me/Projects/connect-extensions/extensions/content-manager/.venv/bin/python
```

Comment:
Can you demo for me on Posit Publisher?

Reply:
Download it for VSCode or Positron. It has a walkthrough which should get you up and running.